lms/envs/common.py

```diff
@@ -421,6 +421,12 @@
 
     # Whether HTML XBlocks/XModules return HTML content with the Course Blocks API student_view_data
     'ENABLE_HTML_XBLOCK_STUDENT_VIEW_DATA': False,
+
+    # Whether the browser alert should be shown or not
+    'ENABLE_UNSUPPORTED_BROWSER_ALERT': False,
+
+    # Sets the default browser support
+    'UNSUPPORTED_BROWSER_ALERT_VERSIONS': "{i:-10,f:-3,o:-3,s:-3,c:-3}",
```

Comment:
@jagonzalr Not `i:-10` please, just `i:10`, since [EdX Browser Support ](http://edx.readthedocs.io/projects/open-edx-learner-guide/en/latest/front_matter/browsers.html) says we support IE 11 and above.